services/src/main/java/io/scalecube/services/Microservices.java

```diff
@@ -309,4 +334,9 @@ public ProxyContext router(Class<? extends Router> router) {
     return servicesTags;
   }
 
+
+  public Transport transport() {
```

Comment:
I just think that this solution is kind of temporary one... so better not to expose via public API transport itself since then later on it will be harder to change stuff if needed, I propose to just expose needed methods there (like send or listen) so then you would have a possibility to internally exchange the implementation without breaking backward compatibility later on.